### PR TITLE
[docs] fix bazel build command

### PR DESCRIPTION
**Pull request by eax-anyscale**
## Problem

The documentation for building ray suggests runnign `bazel build //:*`
This query includes files in addition to targets

## Solution

We should suggest using `//:all` instead. See the [bazel specification](https://bazel.build/query/language#target-patterns) for details

## Test Plan

Documentation! But also I ran it.

### CODE REVIEW

**Review by aslonnie — APPROVED** [review]

File: doc/source/ray-contribute/development.rst [hunk 1]
```diff
@@ -382,7 +382,7 @@ run the following (via ``-c`` ``fastbuild``/``dbg``/``opt``, respectively):
  bazel build -c fastbuild //:ray_pkg
 
 This will rebuild Ray with the appropriate options (which may take a while).
-If you need to build all targets, you can use ``"//:*"`` instead of
+If you need to build all targets, you can use ``"//:all"`` instead of
```
Comment (aslonnie) [inline doc/source/ray-contribute/development.rst:385]: I wonder if it actually means `//...` . `:all` means all targets under the root package.

Comment (aslonnie) [reply doc/source/ray-contribute/development.rst:385]: I don't thin `//...` will work out of the box though..

Comment (eax-anyscale) [reply doc/source/ray-contribute/development.rst:385]: `:*` includes files. `:all` does not.

**Review by krfricke — APPROVED** [review]
Thanks!

**Pull request merged**